client/services/SolutionHandlerService.js

```diff
@@ -19,53 +19,74 @@
 
 tie.factory('SolutionHandlerService', [
   '$q', 'CodePreprocessorDispatcherService', 'CodeRunnerDispatcherService',
-  'FeedbackGeneratorService', 'SnapshotObjectFactory', 'TranscriptService',
-  'CodeSubmissionObjectFactory',
+  'FeedbackGeneratorService', 'PrerequisiteCheckDispatcherService',
+  'SnapshotObjectFactory', 'TranscriptService', 'CodeSubmissionObjectFactory',
```

Comment:
Nit: let's move 'CodeSubmissionObjectFactory' to the line below for parity with line 27. Easier to check that the dependencies line up, that way.